src/Goerli-DssSpell.t.sol

```diff
@@ -479,4 +479,68 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         // assertEq(vest.rxd(1), WAD);
     }
+
+    function testAutoLineGap() public {
+        bytes32 ilk;
+        uint256 line;
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        ilk = "MATIC-A";
+        (,,,line,) = vat.ilks(ilk);
+        assertLe(line, afterSpell.collaterals[ilk].aL_line * RAD);
```

Comment:
if this is before the `exec` is called, shouldn't it be greater than the `aL_line` from the config?

Reply:
I actually think, because this first test is so dependent on chain state, that it doesn't tell us much.  I might just remove it.

Reply:
you could capture the line before the spell and test that the spell doesn't change it then that hte exec does, but it doesn't add a ton of value, so deleting this assert is okay too